Products/CMFPlone/tests/test_okay.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from plone.testing.z2 import Browser
+from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_INTEGRATION_TESTING
+
+import unittest
+
+
+class OkayTest(unittest.TestCase):
+    """Test the OK simple status view.
```

Comment:
Empty line above it? https://www.python.org/dev/peps/pep-0257/#one-line-docstrings says no blank line above or below.